src/types/responses.ts

```diff
@@ -158,3 +158,13 @@ export interface PlayedTileInteractionJSON {
   error: string;
   connectionId: string;
 }
+
+/**
+ * Interface for SELF_PLAY_TILE payload received
+ */
+export interface SelfPlayTileJSON {
+  connectionId: string;
+  playedTile: string;
+  quad?: boolean;
```

Comment:
I changed this to `isQuad` thinking it might be easier to recognize it as a **boolean**

Reply:
Sure !